### DISCUSSION TITLE
company only seems to work by running M-x company-ispell

### DISCUSSION THREAD

**Discussion by dwrz** [company-mode/company-mode#1185] [Q&A] [answered]
I'm trying to get company to provide auto-completion for English, but I'm struggling to get it to work.

I installed company via MELPA, added company-ispell to my backends in customize, and enabled company-mode.

From "Usage" on the homepage:
"Completion will start automatically after you type a few letters." That doesn't seem to be the case for me.

"Type M-x company-complete to initiate completion manually." I get "No completion found."

However, if I run "M-x company-ispell" I get the pop-up with auto-completion. Once I select an option, or finish typing, the pop-up vanishes, and the only way to bring it back for the next word is by running M-x again.

Is there a way to get company-ispell to "stay on"?

**Comment by dgutov** [comment, marked as answer]
From the home page:

> To see or change the list of enabled back-ends, type M-x customize-variable RET company-backends.

**Comment by dwrz** [comment]
Thanks! I'd added ispell there, but missed this: "The choice depends on the order of the items in this list."
